internal/reminder/messages/messages.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2024 Stacklok, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package messages contains the messages used by the reminder service
+package messages
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ThreeDotsLabs/watermill/message"
+	"github.com/go-playground/validator/v10"
+	"github.com/google/uuid"
+)
+
+// RepoReminderEvent is an event that is published by the reminder service to trigger repo reconciliation
+type RepoReminderEvent struct {
+	// Project is the project that the event is relevant to
+	Project uuid.UUID `json:"project"`
+	// RepositoryID is id of the repository to be reconciled
+	RepositoryID int64 `json:"repository" validate:"gte=0"`
```

Comment:
Looking further, it looks like we use the GitHub-matching `repo_id` in NewRepoReconcilerMessage already, so that argues towards using it here as well.

Reply:
Yupp, I used `RepositoryID` to represent GH Id, as Id collided with internal primary key.